### PR TITLE
About me

### PR DESCRIPTION
**Pull request by full-profane**
image

### CODE REVIEW

File: about_me.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About me</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="styles/header_footer.css">
```
Comment (max-si-m) [inline about_me.html]: please, rename `header_footer.css` to base.css as curent name a bit dummy

File: about_me.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <div class="header-menu__item"><i class="fas fa-search header__icon active"></i></div>
+    </div>
+  </header>
+   <div class="about-container lato">
+      <div class="about-content">
+        <p class="about-container__title amatic">about me</p>
+        <div class="about-basis">
+          <img src="img/about-me.jpg">
+          <div class="about-text">
+            <p>Hi, my name is Clare and I’m a serial blogger. I love life and more than anything in the whole wide world, I love taking photographs.</p>
```
Comment (max-si-m) [inline about_me.html:39]: please fix space between lines
telegram-cloud-photo-size-2-5258489316064210834-y

File: about_me.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About me</title>
```
Comment (max-si-m) [inline about_me.html]: change page to `About`

File: styles/itsy.css [hunk 4]
```diff
@@ -30,7 +30,11 @@ header {
   font-size: 25px;
 }
 
-.active, .header-menu__item:hover, .header__icon:hover {
+header a {
```
Comment (max-si-m) [inline styles/itsy.css]: what is it doing here? should be in the `header_footer.css`?

File: styles/about.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    background: #fbfbfb;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    padding: 50px 0 50px 0;
+    color: #8c8a8a;
+}
+
+.lato {
```
Comment (max-si-m) [inline styles/about.css]: should be in the `header_footer.css` as it base for all pages

**Review by max-si-m — CHANGES_REQUESTED** [review]

File: styles/itsy.css (hunk 4 above)
Comment (full-profane) [reply styles/itsy.css]: since here the title is part of the main block, not a stand-alone block.

File: styles/header_footer.css [hunk 6]
```diff
@@ -0,0 +1,137 @@
+* { margin: 0; padding: 0; }
```
Comment (max-si-m) [inline styles/header_footer.css]: please delete this file

File: itsy.html [hunk 7]
```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>ITSY</title>
+  <link rel="stylesheet" type="text/css" href="styles/base.css">
   <link rel="stylesheet" type="text/css" href="styles/itsy.css">
```
Comment (max-si-m) [inline itsy.html]: plese rename `itsy.css` to `index.css` or home page

**Review by max-si-m — CHANGES_REQUESTED** [review]

**Review by max-si-m — APPROVED** [review]

**Pull request merged**